test/fixture-variables/flatten-levels/main.tf

```diff
@@ -5,3 +5,7 @@ data "template_file" "example" {
 output "example" {
   value = "${data.template_file.example.rendered}"
 }
+
+output "example_gotemplate" {
+  value = "@(infra.common.testmap.test1)-@(infra.common.testmap.test2)-@(infra.common.test3)-@(infra.other.hello)"
```

Comment:
This would fail.
The variables would be
`infra.common_testmap_test1` which is completely weird